tests/Fixtures/PathFilters/features/a/feature1.feature

```diff
@@ -0,0 +1,8 @@
+Feature: First Feature
+
+  Scenario: First Scenario
+    Given Some slow step N12
```

Comment:
What are these NX numbers? Do they serve any purpose? They seem weird

Reply:
I just copied the features you guys use:
https://github.com/search?q=repo%3ABehat%2FBehat%20N14&type=code

Reply:
Ah, yeah 🤣 then ignore me